diagnostic.md

```diff
@@ -20,6 +26,8 @@ c = c + d
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d` and explain what they do.
 
+c = 7.  formulas process from right to left. so d=2 and c=5, formula assigns a new value of 7 to c.
+
```

Comment:
The operators we were looking for were `=` to assign value and `+` to add two values.